typescript/creational/builder/builders/NewsRequestBuilder.ts

```diff
@@ -0,0 +1,12 @@
+import {NewsRequest} from "../models/NewsRequest";
+
+export interface NewsRequestBuilderInterface {
+    GenerateRequest(): NewsRequest
+}
+
+export class NewsRequestBuilder implements NewsRequestBuilderInterface {
+
+    GenerateRequest(): NewsRequest {
+        return new NewsRequest()
```

Comment:
If this is where people are expected to start their implementation, add a TODO here with some explicit instructions on what to do - assuming that someone may want to tackle this exercise outside of our workshop at a later time.

Reply:
Updated with instructions